From 2d46230b228db4bbcc12469ad53bdcd693ca38da Mon Sep 17 00:00:00 2001
From: Francesco Ongaro <francesco.ongaro@isgroup.it>
Date: Sat, 24 May 2025 16:22:16 +0200
Subject: [PATCH] Hide menu items in the project dropdown (simpler UI)

---
 app/Template/project_header/dropdown.php | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/app/Template/project_header/dropdown.php b/app/Template/project_header/dropdown.php
index 745ad66..33e2596 100644
--- a/app/Template/project_header/dropdown.php
+++ b/app/Template/project_header/dropdown.php
@@ -18,20 +18,6 @@
                 <i class="fa fa-arrows-h fa-fw"></i> <a href="#" class="filter-toggle-scrolling" title="<?= t('Keyboard shortcut: "%s"', 'c') ?>"><?= t('Horizontal scrolling') ?></a>
             </span>
         </li>
-        <li>
-            <span class="filter-vert-collapse">
-                <i class="fa fa-arrow-up fa-fw"></i> <a href="#" class="filter-vert-toggle-collapse"><?= t('Collapse vertically') ?></a>
-            </span>
-            <span class="filter-vert-expand" style="display: none">
-                <i class="fa fa-arrow-down fa-fw"></i> <a href="#" class="filter-vert-toggle-collapse"><?= t('Expand vertically') ?></a>
-            </span>
-        </li>
-        <?php endif ?>
-
-        <?php if ($this->user->hasProjectAccess('TaskCreationController', 'show', $project['id'])): ?>
-            <li>
-                <?= $this->modal->large('plus', t('Add a new task'), 'TaskCreationController', 'show', array('project_id' => $project['id'])) ?>
-            </li>
         <?php endif ?>
 
         <li>
@@ -75,9 +61,5 @@
                 <?= $this->url->icon('cog', t('Configure this project'), 'ProjectViewController', 'show', array('project_id' => $project['id'])) ?>
             </li>
         <?php endif ?>
-
-        <li>
-            <?= $this->url->icon('folder', t('Manage projects'), 'ProjectListController', 'show') ?>
-        </li>
     </ul>
 </div>
-- 
2.43.0

